extensions/wikia/DesignSystem/services/DesignSystemGlobalFooterService.class.php

```diff
@@ -29,6 +29,10 @@ public function licensingAndVertical() {
 		$this->setVal( 'model', $this->getVal( 'model' ) );
 	}
 
+	public function mobileSiteButton() {
+		$this->setVal( 'model', $this->getVal( 'model' ) );
```

Comment:
you could use `renderPartial` instead when rendering this template, it wont invoke the full dispatcher process and doesn't require having such a boilerplate method defined